### PR TITLE
feat(pie_chart): add pie charts to covalent

### PR DESCRIPTION
**Pull request by aqupriyanka**
### What's included?

- Pie Charts
- Demo For Pie Charts

#### Test Steps

- [ ] `npm run serve`
- [ ] then go to Echarts > Pie

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

##### Screenshots or link to StackBlitz/Plunker
image
image

### CODE REVIEW

File: src/app/components/echarts/types/pie/pie.component.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+              </td-chart>
+            ]]>
+          </td-highlight>
+        </mat-tab>
+      </mat-tab-group>
+    </mat-card-content>
+  </mat-card>
+</div>
+<mat-card>
+  <mat-card-title>Bar (JS Object)</mat-card-title>
```
Comment (thomaslennan) [inline src/app/components/echarts/types/pie/pie.component.html]: This should say Pie?

File: src/platform/echarts/pie/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
++ id?: string
+  + It can be used to refer the component in option or API.
++ name?: string
+  + Series name used for displaying in tooltip and filtering with legend.
++ color?: any
+  + Global color for the series.
++ data?: any[]
+  + Data array of series.
+
+And so many more.. for more info [click here](https://ecomfe.github.io/echarts-doc/public/en/option.html#series-pie)
```
Comment (thomaslennan) [inline src/platform/echarts/pie/README.md]: Should this link work? I got a 404.

**Comment by thomaslennan** [comment]
Noticed a bit of styling issue on the second pie chart. The bottom border doesn't stretch to the end of the screen.

Screen Shot 2019-10-02 at 8 51 13 AM

File: src/app/components/echarts/types/pie/pie.component.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                    </table>
+                  </ng-template>
+                </td-chart-toolbox>
+                <td-chart-tooltip [trigger]="'item'"
+                                  [textStyle]="{ color: '#818181' }"
+                                  [backgroundColor]="'#ffffff'">
+                  <ng-template let-params let-ticket="ticket" tdTooltipFormatter>
+                    <ng-container *ngIf="params">
+                      <div layout="row" layout-align="start center">
+                        <mat-icon>
```
Comment (thomaslennan) [inline src/app/components/echarts/types/pie/pie.component.html]: I think you should drop this icon or make it appear and make it consistent in sample HTML either way.

File: src/app/components/components/components.component.html [hunk 4]
```diff
@@ -24,7 +24,8 @@ <h3 matLine> {{item.title}} </h3>
     </ng-template>
     <h3 matSubheader>Optional Components</h3>
     <a mat-list-item
-        href="https://teradata.github.io/covalent-echarts/"
+        [routerLink]="['/echarts']"
```
Comment (emoralesb05) [inline src/app/components/components/components.component.html]: We should just remove this link

File: src/app/components/echarts/types/pie/pie.component.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+            </td-chart-series>
+          </td-chart>
+        </mat-tab>
+        <mat-tab>
+          <ng-template matTabLabel>Code</ng-template>
+          <p>HTML: </p>
+          <td-highlight lang="html">
+            <![CDATA[
+            <td-chart [style.height.px]="300">
+              <td-chart-dataset [source]=" {
```
Comment (emoralesb05) [inline src/app/components/echarts/types/pie/pie.component.html]: This is missing the end tag for `td-chart-dataset`

**Pull request merged**